physionet-django/physionet/urls.py

```diff
@@ -74,7 +74,7 @@
     ),
 ]
 
-if ProjectFiles().is_lightwave_supported:
+if ProjectFiles().is_lightwave_supported and settings.ENABLE_LIGHTWAVE:
```

Comment:
```suggestion
if settings.ENABLE_LIGHTWAVE and ProjectFiles().is_lightwave_supported:
```

wdyt? Now the hierarchy is clearer:
1. Is this feature even supported?
2. If it is, is it supported by our other configuration?

Reply:
makes sense